### PR TITLE
Release | Release notes for v2.1.2

### PR DESCRIPTION
**Pull request by cheenamalhotra**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,21 @@ All notable changes to this project will be documented in this file.
...
+- Fixed issue connecting with instance name from Unix environment [#874](https://github.com/dotnet/SqlClient/pull/874)
+- Fixed wrong data issues by reworking timeout timer to ensure it's guaranteed execution [#929](https://github.com/dotnet/SqlClient/pull/929)
+- Fixed vulnerability by prohibiting `DtdProcessing` on `XmlTextReader` instance in .NET Core [#885](https://github.com/dotnet/SqlClient/pull/885)
+- Fixed Kerberos authentication when SPN did not contain port [#935](https://github.com/dotnet/SqlClient/pull/935)
+- Fixed missing error messages in Managed SNI [#883](https://github.com/dotnet/SqlClient/pull/883)
+- Fixed missing `System.Runtime.Caching` dependency for .NET Standard assemblies [#878](https://github.com/dotnet/SqlClient/pull/878)
+- Fixed event source tracing issues [#941](https://github.com/dotnet/SqlClient/pull/941)
+
+### Changes
+- Enabled Read Async blocking for all supported .NET Framework versions by default [#928](https://github.com/dotnet/SqlClient/pull/928)
```
Comment (David-Engel) [inline CHANGELOG.md:19]: I think this should go under fixed issues as it was not intended to be disabled.

"Fixed MARS header contains errors issue against .NET Framework 4.8.1" [link to PR]

File: release-notes/2.1/2.1.2.md [hunk 2]
```diff
@@ -0,0 +1,85 @@
+# Release Notes
+
+## Microsoft.Data.SqlClient 2.1.2 released 3 March 2021
+
+This update brings the below changes over the previous stable release:
+
+### Fixed
+- Fixed issue connecting with instance name from Unix environment [#874](https://github.com/dotnet/SqlClient/pull/874)
```
Comment (David-Engel) [inline release-notes/2.1/2.1.2.md:8]:
```suggestion
- Fixed issue connecting with instance name from a Linux/macOS environment [#874](https://github.com/dotnet/SqlClient/pull/874)
```

File: release-notes/2.1/2.1.2.md [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+- Fixed wrong data issues by reworking timeout timer to ensure it's guaranteed execution [#929](https://github.com/dotnet/SqlClient/pull/929)
```
Comment (David-Engel) [inline release-notes/2.1/2.1.2.md:9]:
```suggestion
- Fixed wrong results issues by changing the timeout timer to ensure a correct execution state [#929](https://github.com/dotnet/SqlClient/pull/929)
```

File: release-notes/2.1/2.1.2.md [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+- Fixed vulnerability by prohibiting `DtdProcessing` on `XmlTextReader` instance in .NET Core [#885](https://github.com/dotnet/SqlClient/pull/885)
```
Comment (David-Engel) [inline release-notes/2.1/2.1.2.md:10]:
```suggestion
- Fixed a vulnerability by prohibiting `DtdProcessing` on `XmlTextReader` instances in .NET Core [#885](https://github.com/dotnet/SqlClient/pull/885)
```

File: release-notes/2.1/2.1.2.md [hunk 5]
```diff
@@ -0,0 +1,85 @@
...
+- Fixed Kerberos authentication when SPN did not contain port [#935](https://github.com/dotnet/SqlClient/pull/935)
```
Comment (David-Engel) [inline release-notes/2.1/2.1.2.md:11]:
```suggestion
- Fixed Kerberos authentication when an SPN does not contain the port [#935](https://github.com/dotnet/SqlClient/pull/935)
```

File: release-notes/2.1/2.1.2.md [hunk 6]
```diff
@@ -0,0 +1,85 @@
...
+- Fixed missing error messages in Managed SNI [#883](https://github.com/dotnet/SqlClient/pull/883)
+- Fixed missing `System.Runtime.Caching` dependency for .NET Standard assemblies [#878](https://github.com/dotnet/SqlClient/pull/878)
+- Fixed event source tracing issues [#941](https://github.com/dotnet/SqlClient/pull/941)
+
+### Changes
+- Enabled Read Async blocking for all supported .NET Framework versions by default [#928](https://github.com/dotnet/SqlClient/pull/928)
```
Comment (David-Engel) [inline release-notes/2.1/2.1.2.md:17]: Same comment as above.

File: CHANGELOG.md (hunk 1 above)
Comment (cheenamalhotra) [inline CHANGELOG.md:19]:
```suggestion
- Fixed MARS header contains errors issue against .NET Framework 4.8.1 [#928](https://github.com/dotnet/SqlClient/pull/928)
```

File: release-notes/2.1/2.1.2.md (hunk 6 above)
Comment (cheenamalhotra) [inline release-notes/2.1/2.1.2.md:17]:
```suggestion
- Fixed MARS header contains errors issue against .NET Framework 4.8.1 [#928](https://github.com/dotnet/SqlClient/pull/928)

```

**Review by David-Engel — APPROVED** [review]

**Pull request merged**